docs/dev/reference/hooks/newsListFetchItems.md

```diff
@@ -69,4 +71,4 @@ class NewsListFetchItemsListener
 
 ## References
 
-* [\Contao\ModuleNewsList#L200-L215](https://github.com/contao/contao/blob/4.7.6/news-bundle/src/Resources/contao/modules/ModuleNewsList.php#L200-L215)
+* [\Contao\ModuleNewsList#L205-L220](https://github.com/contao/contao/blob/5.3/news-bundle/contao/modules/ModuleNewsList.php#L205-L220)
```

Comment:
```suggestion
* [\Contao\ModuleNewsList#L205-L220](https://github.com/contao/contao/blob/de632fca83b601bd8496058e3f8a8e874ad8c341/news-bundle/contao/modules/ModuleNewsList.php#L205-L220)
```

Otherwise this link will point to the wrong location or result in 404, if either code is changes or the branch gets deleted.